### PR TITLE
version bump to 1.3.9

### PR DESCRIPTION
**Pull request by jordarlu**
### Description
version bump to 1.3.9
 
### Issues Resolved
post task for release 1.3.9
https://github.com/opensearch-project/opensearch-build/issues/3195
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [x] Helm chart version bumped
- [x] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by prudhvigodithi — APPROVED** [review]

**Review by TheAlgo — CHANGES_REQUESTED** [review]
@jordarlu Can you please attach an issue to this PR?

**Comment by prudhvigodithi** [comment]
Hey @TheAlgo thanks I have updated the description, please check.

File: charts/opensearch-dashboards/CHANGELOG.md [hunk 1]
```diff
@@ -350,6 +359,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 
 [Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-1.11.0...HEAD
```
Comment (gaiksaya) [inline charts/opensearch-dashboards/CHANGELOG.md:361]: Should be 1.12.0...HEAD

File: charts/opensearch/CHANGELOG.md [hunk 2]
```diff
@@ -545,6 +554,7 @@ config:
 ### Security
 
 [Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-1.18.0...HEAD
```
[the remark this answers is not in the document]
Comment (jordarlu) [reply charts/opensearch/CHANGELOG.md:556]: Thanks for the correction on where I miseed ... updating it now @gaiksaya and @prudhvigodithi ... thanks!

[the remark this answers is not in the document]
Comment (gaiksaya) [reply charts/opensearch/CHANGELOG.md:556]: Missed this one.

[the remark this answers is not in the document]
Comment (jordarlu) [reply charts/opensearch/CHANGELOG.md]: :( I did miss this one ...

**Review by gaiksaya — APPROVED** [review]

**Comment by gaiksaya** [comment]
Please rebase to 1 commit @jordarlu Thanks!

**Comment by prudhvigodithi** [comment]
Hey @TheAlgo can you please check this PR again, Not able to merge as it as `requested changes` from you.
Thank you

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**